src/mprisplayer.cpp

```diff
@@ -66,7 +67,17 @@ MprisPlayer::MprisPlayer(QObject *parent)
     , m_shuffle(false)
     , m_volume(0)
 {
+    if (parent != NULL) {
+        if (parent->property("useSystemBus").isValid()) {
+            qmlInfo(this) << "MprisPlayer using system bus";
+            m_dbusType = QDBusConnection::SystemBus;
```

Comment:
Commented elsewhere already, but this feels like a thing that should be passed on the ctor parameters.